src/Lykke.Service.PayInternal.Services/BtcTransferRequestService.cs

```diff
@@ -61,7 +66,7 @@ public BtcTransferRequestService(ITransferRepository transferRepository, IWallet
                 transferRequest.DestinationAddress
             };
 
-            var isAddressesValid = await CheckAddressesValid(transferWallets);
+            var isAddressesValid = CheckAddressesValid(transferWallets);
             if (!isAddressesValid)
             {
                 return new TransferDto
```

Comment:
I'd make TransferDto factory to create this kind of objects and not to pollute the BL code.